pallets/fragments/src/lib.rs

```diff
@@ -1922,6 +1922,36 @@ pub mod pallet {
 			Ok(())
 
 		}
+
+		/// Query the owner of a Fragment Instance. The return type is AccountId
+		pub fn get_instance_owner_account_id(
+			params: GetInstanceOwnerParams<Vec<u8>>,
+		) -> Result<T::AccountId, Vec<u8>> {
+			let definition_hash: Hash128 = hex::decode(params.definition_hash)
```

Comment:
see here also you do a conversion...

Reply:
from hex???

Reply:
I did not change this. This is how it was implemented before.

Reply:
Yes and it's correct in an RPC CONTEXT! Json, hexes etc, right?